CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.16.0 Release
+- Release Date: January 16, 2019
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Combatibility
+
+ - Mobile App v1.13+ is required for Mattermost Server v5.4+.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+
+### Bug Fixes
```

Comment:
We should sort these in rough order of severity in my mind (at least for the more severe bugs), ie...: